README.md

```diff
@@ -1 +1,13 @@
-# building-a-checks-api-ci-server
\ No newline at end of file
+This is the sample project he built by following the "[Building Your First GitHub App](https://developer.github.com/apps/building-your-first-github-app)" Quickstart guide on developer.github.com.
```

Comment:
Seems there's a typo in the first sentence here. Should this be:

> This is the sample project built by following

Or even in the passive:

> Build this sample project by following